src/components/EarnReport.tsx

```diff
@@ -158,27 +159,47 @@ export function EarnReport() {
           {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
           {yieldgenReportLines && (
             <rb.Row>
-              <rb.Col className="mb-3">
-                <YieldgenReportTable lines={yieldgenReportLines} />
+              <rb.Col>
+                <div className={styles.earnReportContainer}>
+                  <YieldgenReportTable lines={yieldgenReportLines} />
+                </div>
               </rb.Col>
             </rb.Row>
           )}
         </>
       )}
-    </div>
+    </>
   )
 }
 
 export function EarnReportOverlay({ show, onHide }: rb.OffcanvasProps) {
   const { t } = useTranslation()
 
   return (
-    <rb.Offcanvas className={styles['report-overlay']} show={show} onHide={onHide} placement="bottom">
-      <rb.Offcanvas.Header closeButton>
-        <rb.Offcanvas.Title>{t('earn.report.title')}</rb.Offcanvas.Title>
+    <rb.Offcanvas
+      className={`offcanvas-fullscreen ${styles.overlayContainer}`}
+      show={show}
+      onHide={onHide}
+      placement="bottom"
+    >
+      <rb.Offcanvas.Header>
+        <rb.Container>
+          <div className="w-100 d-flex">
+            <div className="d-flex align-items-center flex-1">
+              <rb.Offcanvas.Title>{t('earn.report.title')}</rb.Offcanvas.Title>
+            </div>
+            <div>
+              <rb.Button variant="link" className="unstyled pe-0 ms-auto me-auto me-md-0" onClick={onHide}>
```

Comment:
```suggestion
              <rb.Button variant="link" className="unstyled pe-0" onClick={onHide}>
```

I don't think you need these margins. The enclosing div will always be just as large as the cancel button. It has no room to grow as the other element in the flexbox is `flex-1` but itself is not. So its child doesn't need those margins.

The reason why they're needed in the Jar Details header is because there the enclosing div of the cancel button _can_ grow (it's `flex-1`). Therefore we need to push the cancel button to the right with `ms-auto`. Also, there the entire header breaks to a `flex-column` on mobile so we need the `me-auto` to center it in this case.

```
 JAR DETAILS LARGE SCREEN
+-------------------------------------------------------------------------+
|                                                                         |
|   +-----------------+ +------------------------+ +------------------+   |
|   |                 | |                        | |                  |   |
|   |                 | |                        | |   ms-auto  +---+ |   |
|   |                 | |                        | | <--------- | X | |   |
|   |                 | |                        | |            +---+ |   |
|   |                 | |                        | |                  |   |
|   +-----------------+ +------------------------+ +------------------+   |
|                                                                         |
+-------------------------------------------------------------------------+


 JAR DETAILS SMALL SCREEN
+---------------------------------------+
|                                       |
|   +-------------------------------+   |
|   |                               |   |
|   |                               |   |
|   +-------------------------------+   |
|                                       |
|   +-------------------------------+   |
|   |                               |   |
|   |                               |   |
|   +-------------------------------+   |
|                                       |
|   +-------------------------------+   |
|   |                               |   |
|   |     ms-auto  +---+  me-auto   |   |
|   |  <---------- | X | ---------> |   |
|   |              +---+            |   |
|   |                               |   |
|   +-------------------------------+   |
|                                       |
+---------------------------------------+



 EARNDREPORT / ORDERBOOK
+-------------------------------------------------------------------------+
|                                                                         |
|   +-----------------------------------------------------+ +---------+   |
|   |                                                     | |         |   |
|   |                                                     | |  +---+  |   |
|   |                                                     | |  | X |  |   |
|   |                                                     | |  +---+  |   |
|   |                                                     | |         |   |
|   +-----------------------------------------------------+ +---------+   |
|                                                                         |
+-------------------------------------------------------------------------+
```

Reply:
Thanks for the detailed explanation. Highly appreciated. :pray: